src/opera/pge/disp_s1/disp_s1_pge.py

```diff
@@ -609,6 +640,22 @@ def _collect_disp_s1_product_metadata(self, disp_product):
         # Extract all metadata assigned by the SAS at product creation time
         try:
             output_product_metadata = get_disp_s1_product_metadata(disp_product)
+
+            # get_catalog_metadata_datetime_str(self.production_datetime)
+
+            # Add hardcoded values to metadata
+            output_product_metadata['static'] = {
+                'Project': 'OPERA',
+                'ProductLevel': 3,
+                'ProductType': 'DISP-S1',
+                'ProductSource': 'Sentinel-1',
+                'ProcessingDateTime': get_catalog_metadata_datetime_str(self.production_datetime)
+            }
+
+            output_product_metadata['MeasuredParameters'] = self.augment_measured_parameters(output_product_metadata)
+
+            # Remove the hardcoded dict from main output product metadata dict once it's served its purpose
+            del output_product_metadata['static']
```

Comment:
Is this really necessary? Does leaving it in cause unintended side-effects? If so, would be good to know that in the comment here.

Reply:
I didn't really think it would be, I was thinking to minimize changes to that dictionary just in case there would be side-effects. I removed it for now

Reply:
Ran a diff and there's no issues with this statement gone